snapcraft.yaml

```diff
@@ -582,10 +582,11 @@ parts:
       - libseccomp
     source: https://github.com/NVIDIA/libnvidia-container
     source-commit: 4c2494f16573b585788a42e9c7bee76ecd48c73d # v1.16.1
-    # XXX: fails to build if using source-commit and source-depth
-    #source-depth: 1
+    source-depth: 1
     source-type: git
     plugin: make
+    build-environment:
+      - GIT_TAG: "1.16.1" # Enables source-depth: 1, should match git tag without "v" prefix.
```

Comment:
I wonder if it wouldn't be feasible to use `GIT_TAG="$(git describe --tags --abbrev=0 | sed 's/^v//')"` in the override-build?

Reply:
I initially tried doing `export GIT_TAG="1.16.1"` and it didnt work when using `snapcraftctl build` so looks like it resets the env

Reply:
yeah thats right it was there

Reply:
before the call to `snapcraftctl build`